src/cdk/tree/tree.ts

```diff
@@ -304,12 +308,21 @@ export class CdkTree<T> implements CollectionViewer, OnInit, OnDestroy {
    * Create the embedded view for the data node template and place it in the correct index location
    * within the data node view container.
    */
-  insertNode(nodeData: T, index: number, viewContainer?: ViewContainerRef) {
+  insertNode(nodeData: T, index: number, viewContainer?: ViewContainerRef, parentData?: T) {
     const node = this._getNodeDef(nodeData, index);
 
     // Node context that will be provided to created embedded view
     const context = new CdkTreeNodeOutletContext<T>(nodeData);
 
+    if (this.treeControl.getLevel) {
```

Comment:
Add comment that explains under which circumstances `getLevel` would be defined or not?